h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -205,10 +225,16 @@ boolean isUsefulBaseModel(Key<Model> baseModelKey) {
   }
 
   ModelMetrics doScoreMetricsOneFrame(Frame frame, Job job) {
-      Frame pred = this.predictScoreImpl(frame, new Frame(frame), null, job, true, CFuncRef.from(_parms._custom_metric_func));
-//      Frame pred = this.score(frame, null, job, true,  CFuncRef.from(_parms._custom_metric_func));
+    Frame scoredFrame = (_parms._score_training_samples > 0 && _parms._score_training_samples < frame.numRows())
+            ? MRUtils.sampleFrame(frame, _parms._score_training_samples, _parms._seed)
+            : frame;
+    try {
+      Frame pred = this.predictScoreImpl(scoredFrame, new Frame(scoredFrame), null, job, true, CFuncRef.from(_parms._custom_metric_func));
```

Comment:
This would be a use case for `Model#scoreMetrics` function. It doesn't have the custom metric function though - I don't think you care about the custom metrics here, WDYT?

Reply:
@michalkurka For now, we don't support/need custom metrics.  In the future, we might, but we would change it then.

Reply:
@tomasfryda the switch to `scoreMetrics` can be done in a separate ticket and addressed later as it's not related with your current changes. Feel free to file one in Jira.

Reply:
Ok, I created it: https://0xdata.atlassian.net/browse/PUBDEV-7478